docs/content/topics/scripting/workers.md

```diff
@@ -9,216 +9,117 @@ aliases:
-| `allowPrivilegedJobs` | A boolean (represented as the _string_ `"true"` or `"false"`) indicating whether pods that implement each build's job(s) may include privileged containers. | |
-| `buildStorageSize` | The desired size for any shared build storage and build cache volumes that are provisioned. | |
-| `initGitSubmodules` | If applicable, a boolean (represented as the _string_ `"true"` or `"false"`) indicating whether any git submodules should be initialized after project source is retrieved from VCS. | |
-| `kubernetes.buildStorageClass` | Specifies the desired Kubernetes storage class to be used for any shared build storage volume that is provisioned. | This can override the Brigade-level default. |
-| `kubernetes.cacheStorageClass` | Specifies the desired Kubernetes storage class to be used for any build cache volume that is provisioned. | This can override the Brigade-level default. |
-| `secrets` | Base64-encoded JSON containing project-specific secrets. | |
-| `vcsSidecar` | If applicable, image to be used by "VCS sidecar" containers that obtain project source code from a VCS repository. | |
+worker as well as the methods whereby Brigade passes configuration to a worker
+for processing an event.
+
+[Dependencies]: /topics/scripting/dependencies
+
+## Event Data
+
+Workers get the data they need to process an event via JSON mounted to its pod
+by Brigade, each time it runs. This JSON contains configuration that the worker
+itself uses, as well as project- and event-specific details. This file is
+located at `/var/event/event.json` inside the worker container.
+
+Note that custom workers may selectively disregard any data they deem
+inapplicable to the custom behavior they implement. For instance, a field that
+conveys the expected location of the `brigade.js` file is not applicable to a
+worker that does not utilize such configuration.
+
+### Fields
+
+Here's a rundown on the data present in the event JSON:
+
+| Field Name | Description |
+|------------|-------------|
+| `id` | A unique ID for the event |
+| `project.id` | The project ID that this event is associated with |
+| `project.kubernetes.namespace` | The Kubernetes namespace that the project uses |
+| `project.secrets` | The key/value map of [secrets] for the project |
+| `source` | The event [source] |
+| `type` | The event [type] |
+| `qualifiers` | The event [qualifiers] |
+| `labels` | The event [labels] |
+| `shortTitle` | A [short title] for the event |
+| `longTitle` | A [long title] for the event |
+| `payload` | The event [payload] |
+| `worker.apiAddress` | The address for the Brigade API server |
+| `worker.apiToken` | The token used for communicating with the Brigade API server |
+| `worker.logLevel` | The log level for worker logs |
+| `worker.configFilesDirectory` | The directory in the worker where the Brigade config files may be found |
```

Comment:
Should probably note that this is applicable when there's a git repo associated with the project and the path is relative to where the source code from that repo has already been mounted in the worker's container (`/var/vcs`).